### DISCUSSION TITLE
Can I change the label of the btn of specific excel export ??

### DISCUSSION THREAD

**Discussion by Alsanea-Ala** [pxlrbt/filament-excel#162] [Q&A] [answered]
![Screenshot_20231225_093130](https://github.com/pxlrbt/filament-excel/assets/88606691/7f3a07c6-e030-4771-ad7b-1b38dc3329e3)

for example I want to make export for specific columns and all columns !!

**Comment by Alsanea-Ala** [comment, marked as answer]
I just add ->lable('name') after ->exports() and it works 🙂🙂🙂

**Comment by Alsanea-Ala** [comment]


**Reply by pxlrbt** [reply]
I don't understand the issue?
